Kerdy/Kerdy/MyProfile/ProfileTagBtn.swift

```diff
@@ -74,7 +87,7 @@ final class ProfileTagBtn: UIStackView {
         borderLayer.strokeColor = UIColor(named: "kerdy_main")?.cgColor
         borderLayer.strokeColor = UIColor.kerdyMain.cgColor
```

Comment:
```suggestion
        borderLayer.strokeColor = UIColor.kerdyMain.cgColor
```
윗 줄은 삭제해도 좋을 것 같습니다!